scripts/generate_dump

```diff
@@ -536,17 +711,59 @@ collect_mellanox() {
 #  None
 ###############################################################################
 collect_broadcom() {
-    save_cmd "bcmcmd -t5 version" "broadcom.version"
-    save_cmd "bcmcmd -t5 soc" "broadcom.soc"
-    save_cmd "bcmcmd -t5 ps" "broadcom.ps"
+    local platform="$(/usr/local/bin/sonic-cfggen -H -v DEVICE_METADATA.localhost.platform)"
+    local hwsku=$(head -n 1 /usr/share/sonic/device/${platform}/default_sku | cut -d ' ' -f 1)
+
+    # save SAI configuration files (config.bcm, port_config.ini, sai.profile)
+    if [ -d /usr/share/sonic/device/${platform}/${hwsku} ]; then
+        # copy all the files in the HWSKU directory
+        pushd /usr/share/sonic/device/${platform}/${hwsku} > /dev/null
+        for file in $(find . -maxdepth 1 -type f); do
```

Comment:
change maxdept to '2' to include files for multi ASIC platforms as their file location is under ASIC ID subdir.

/usr/share/sonic/device/${platform}/${hwsku}/<asicID>

Reply:
Done

Reply:
done